DEVELOPMENT.md

```diff
@@ -1,135 +1,73 @@
-# Development
+Development
+This doc explains how to set up a development environment so you can get started contributing to Knative Eventing. Al l so take a look at the development workflow and the test docs.
```

Comment:
I'm not sure what's going on here, but this reformatting seems like it has lost all the markdown formatting (and run together a bunch of lines).

https://github.com/knative/eventing-sources/blob/8ee5072f202c3d04ae5ae1d4bec7d2f472a2e96f/DEVELOPMENT.md